update.sh

```diff
@@ -14,7 +14,14 @@ declare -A debianVariants=(
 	#[5.5]='jessie'
 )
 
+# Copy entrypoint template
+templateVersions=( "5.7 8.0" )
+for version in ${templateVersions}; do
+	cp "template.Debian/docker-entrypoint.sh" "${version}/"
```

Comment:
There's really not a _whole_ lot that's different between these and 5.5/5.6 -- do you think it'd make sense to do some runtime detection and switch behavior in the scripts dynamically in order to hit all four versions?  We could also have this script include the nuances for the lower versions and have it dynamically update the template when copying it to the lower versions.

Reply:
Yeah, you're right it's not that much work (we do this at https://github.com/mysql/mysql-docker/blob/mysql-server/gen_dockerfiles.sh).